### DISCUSSION TITLE
Theme Viewer for 4.0?

### DISCUSSION THREAD

**Discussion by ChilledEther** [wixtoolset/issues#7457] [Questions] [answered]
I can't seem to find an answer to this one, but is there a new Theme Viewer for 4.0 rather than having to build installer for every test?

Thanks!

**Comment by robmen** [comment, marked as answer]
It's part of the "WixAdditionalTools.exe" bundle you can find in the `wix` GitHub releases or via winget.

**Comment by ChilledEther** [comment]
Ah perfect, as easy as that. Tested and works fine!
